scripts/liftoff.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# First create new repo on GitHub
+# Then run the following script from within local hpp-skel root directory:
+#
+# liftoff.sh --name <new-name> --url <new-remote-repo-url> 
```

Comment:
This is not passing arguments yet, right? How about using `read` to interactively prompt for arguments? (rather than accepting on the command line). The reason I suggest this is:

1. parsing args from the command line is somewhat painful. https://github.com/mapbox/hpp-skel/blob/1afb4bf51b4bf36582c0419987afed97c33bf15a/scripts/setup.sh#L113-L132 is an example but it is hard to write

2. using `read` gives the use the opporunity to provide docs per option in the terminal, gives the user an opportunity to think about it and cancel out if they don't know what to provide.

Read example at https://stackoverflow.com/a/226720

Reply:
This is super cool, didnt know about `read`. Also apologies on the lazy PR, this was super rough and wanted to get it up before heading out yesterday. Appreciate your feedback 👍